paasta_tools/setup_kubernetes_job.py

```diff
@@ -82,6 +73,7 @@ def parse_args() -> argparse.Namespace:
 
 
 def main() -> None:
+    global setup_kube_succeeded
```

Comment:
how come the change to a global? I'd prefer to keep the return values if it's not too tricky